src/pudl/analysis/plant_parts_eia.py

```diff
@@ -367,32 +367,30 @@
 
 
 @asset(
-    name="mega_generators_eia",
     compute_kind="Python",
 )
-def mega_gens_asset(
-    mcoe_generators_yearly: pd.DataFrame, out_eia860__yearly_ownership: pd.DataFrame
+def out_eia__yearly_generators_by_ownership(
+    out_eia__yearly_generators: pd.DataFrame, out_eia860__yearly_ownership: pd.DataFrame
 ) -> pd.DataFrame:
     """Create mega generators table asset."""
     return MakeMegaGenTbl().execute(
-        mcoe=mcoe_generators_yearly,
+        mcoe=out_eia__yearly_generators,
         own_eia860=out_eia860__yearly_ownership,
     )
 
 
 @asset(
-    name="plant_parts_eia",
     io_manager_key="pudl_sqlite_io_manager",
     compute_kind="Python",
 )
-def plant_parts_eia_asset(
-    mega_generators_eia: pd.DataFrame,
+def out_eia__plant_parts(
```

Comment:
asset type can be optional. I didn't think there was logical frequency or type for the table though I could be wrong. @cmgosnell and @katie-lamb what do you think?

Reply:
I believe the plant parts are assigned annually, but correct me if I'm wrong @cmgosnell

Reply:
it's an annual table! (it technically could be generated as a monthly table but it would be ginombo and rn this is mostly being used to link up to annual ferc data)

Reply:
thanks! It has been renamed to `out_eia__yearly_plant_parts`.